core/src/miner/miner.rs

```diff
@@ -374,6 +374,11 @@ impl Miner {
         results
     }
 
+    pub fn delete_all_transactions(&self) {
```

Comment:
What do you think about using `delete_all_pending_transactions` as name instead?

Reply:
Ok